de_docs/getting-started/beginner-guide.md

```diff
@@ -41,7 +41,7 @@ To get started, choose an option for installations available for your operating
 
 * [Paymetheus](/getting-started/install-guide.md#paymetheus)
 * [Command Line Suite](/getting-started/install-guide.md#dcrinstall)
-* [Decrediton (BETA)](https://github.com/decred/decrediton) - NOTE: Install instructions currently limited to the Github README.
+* [Decrediton (BETA)](https://github.com/decred/decrediton) - NOTE: Install instructions currently limited to the GitHub README.
```

Comment:
I just rebased master, but it didn't result in any merge conflicts. Can you point out a specific commit to master that this line conflicts with? Or is it just a matter of two different PRs touching the same line?